jmxtrans-cluster/src/main/java/com/googlecode/jmxtrans/cluster/zookeeper/ZookeeperConfig.java

```diff
@@ -0,0 +1,69 @@
+package com.googlecode.jmxtrans.cluster.zookeeper;
+
+import com.googlecode.jmxtrans.cluster.ClusterService;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * ZookeeperConfig. This class stores the Zookeeper related configuration parts.
+ *
+ * @author Tibor Kulcsar
+ * @since <pre>May 17, 2016</pre>
+ */
+public class ZookeeperConfig {
+    public static final String OWNER_NODE_NAME  = "owner";
+    public static final String AFFINITY_NODE_NAME = "affinity";
+    public static final String CONFIG_NODE_NAME = "config";
+    public static final String REQUEST_NODE_NAME = "request";
+
+    @Getter @Setter private String connectionString;
```

Comment:
Are you sure we need to have `@Setter` here? It seems that we could make this class immutable (and tag it as such with `@Immutable`).